run-some-asp-tests

```diff
@@ -16,6 +16,6 @@ testcases='m3ua-asp-aspsm-v-002
 
 for testcase in $testcases
 do
-  runm3uatest -t $timeout $testcase 2> /dev/null
+  ./runm3uatest -d . -t $timeout $testcase 2> /dev/null
```

Comment:
This requires the binary to be in the directory where the script is. I assume that this is installed, which is also the case when running the tests manually.

Reply:
requiring installation of a helper script to some system PATH seems like a relatively odd choice for a test tool.  99.9% of the FOSS software I know of can be executed from within the checked out git repo / build directory, without requiring system-wide installation.  I see that a s a strong benefit, avoiding to clutter my system usr/local/bin etc. with tons of cruft over time.

You're of course free to reject it.  With the python-based altrnative executor of the other patch we at osmocom don't even use runm3uatest anymore.  It was just something developed at an earlier stage trying to make the nplab-tests run for the first time some 4 years ago.